src/Raygun4php/RaygunClient.php

```diff
@@ -372,116 +366,17 @@ private function is_assoc($array)
     }
 
     /**
-     * Transmits a RaygunMessage to the Raygun.io API. The default attempts to transmit asynchronously.
-     * To disable this and transmit sync (blocking), pass false in as the 2nd parameter in RaygunClient's
-     * constructor. This may be necessary on some Windows installations where the implementation is broken.
+     * Transmits a RaygunMessage to the Raygun.io API.
      * This is a lower level function used by SendException and SendError and one of those should be used preferably.
      *
      * @param \Raygun4php\RaygunMessage $message A populated message to be posted to the Raygun API
-     * @return int|null The HTTP status code of the result after transmitting the message to Raygun.io
-     *                                          202 if accepted, 403 if invalid JSON payload
-     * @throws Raygun4PhpException
+     * @return bool Returns true if the transmission attempt is successful.
+     *              However, this does not guarantee that the message is delivered.
```

Comment:
Does this mean that it can return true when it has not delivered the data? If it does return true and the message has not been delivered, is there any way to know this (e.g. logging)?

Reply:
Yes, you could have a case where some async implementation of `TransportInterface
` successfully cues a message for delivery and returns True, but the actual delivery fails for whatever reason. The natural place to log such an event would be in the `transport` implementation.